### PR TITLE
adds new relic to Editor client side

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

File: newrelic.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['Performance Dashboard - Editor'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: process.env.NEW_RELIC_LICENSE_KEY,
```
Comment (elisechant) [inline newrelic.js]: @micapam assume that NEW_RELIC_LICENSE_KEY is a prop of .env in the live environment, like it's accessed here - newrelic.yml:15 ?

File: newrelic.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * New Relic agent configuration.
+ *
+ * See lib/config.defaults.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['Performance Dashboard - Editor'],
```
Comment (elisechant) [inline newrelic.js]: @micapam are you happy with this app name?

File: newrelic.js (hunk 1 above)
Comment (tobyhede) [inline newrelic.js]: Yep. Present in both staging and production. Reports generated for both in NR.

**Comment by elisechant** [comment]
@micapam, going to merge this now.

**Pull request closed without merging**